app.rb

```diff
@@ -106,7 +114,7 @@ def add_rental
 
     print 'Date: '
     date = get_user_input('')
-
+    LibraryManager.write_rentals(rentals)
     handle_rental_creation(date, book, person)
   end
 
```

Comment:
- You have done well so far, but your application throws an error when adding a new rental. Please note that your application should work as expected without bugs, please fix this. The error is coming from the logic for storing data. Please fix this in your `storage.rb` file 👍 👍 
![fhfhfhf](https://github.com/PrinceBadru/OOP-school-library/assets/73607512/82e8b54f-b253-4a44-bf4e-c85e58dda95c)